<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerminalMainWindow</class>
 <widget class="QMainWindow" name="TerminalMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Marlene Terminal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="recv_addr_disp">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Receiving Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="acct_balance_disp">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Account Balance</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txfee_disp">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Transaction Fee</string>
    </property>
   </widget>
   <widget class="QPushButton" name="query_acct_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Account Details</string>
    </property>
   </widget>
   <widget class="QPushButton" name="make_payment_btn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Make Payment</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="payto_addr_edit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pay to address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="amount_edit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pin_edit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>9999</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PIN</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Terminal"/>
    <addaction name="actionAbout_Card"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <action name="actionAbout_Terminal">
   <property name="text">
    <string>About Terminal</string>
   </property>
  </action>
  <action name="actionAbout_Card">
   <property name="text">
    <string>About Card</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>query_acct_btn</sender>
   <signal>clicked()</signal>
   <receiver>TerminalMainWindow</receiver>
   <slot>onGetAccountDetails()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>make_payment_btn</sender>
   <signal>clicked()</signal>
   <receiver>TerminalMainWindow</receiver>
   <slot>onMakePayment()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onGetAccountDetails()</slot>
  <slot>onMakePayment()</slot>
 </slots>
</ui>
